README.md

```diff
@@ -74,15 +74,17 @@ Total Flash memory (text + data + misc): 60554 bytes
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+It creates two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory. The original unpadded image is saved with the name `<project-name>_application.bin`. The padded bootloader image (used for combining with applications) is saved with the name `<project-name>.bin`.
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+When the build succeeds, you have created a bootloader for your target. This bootloader defines the update binary to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin` on the sd card. This is the binary that the bootloader flashes and then jumps to.
+
+<span class="tips">**Note:** The `/sd/` part of the update binary path **does not** correspond to a directory of `sd` on the SD card's filesystem. The entire SD card is mounted to the filesystem as the directory `/sd/`. For this example, you should place the update binary in the root directory of the SD card.</span>
 
 ## Set up application to use bootloader
 
-The next step is to build an application you can combine with your bootloader to create a loadable image. 
+The next step is to build an application you can combine with your bootloader to create a loadable image. Any application should work, however you may wish to start with the [blinky example](https://github.com/ARMmbed/mbed-os-example-blinky).
 
-1. Update your board to use the newly created bootloader image. To do this, set the target (replace `<TARGET_NAME>` with your target name) value `bootloader_img` to the file path of the bootloader image.
+1. Instruct the build system to use the newly created padded bootloader image (named `<project-name>.bin` in the previous section). To do this, set the target value `bootloader_img` to the file path of the bootloader image (replace `<TARGET_NAME>` with your target name). In the below example, the bootloader image has been copied to a new directory called `bootloader` in the project and renamed to `<TARGET_NAME>.bin`.
```

Comment:
Query: Who or what has copied the bootloader image to the new directory and renamed it? Please clarify.